internal/conf/conf.go

```diff
@@ -39,12 +39,17 @@ type tlsConf struct {
 }
 
 type SinkConf struct {
-	MemoryCacheThreshold int  `json:"memoryCacheThreshold" yaml:"memoryCacheThreshold"`
-	MaxDiskCache         int  `json:"maxDiskCache" yaml:"maxDiskCache"`
-	BufferPageSize       int  `json:"bufferPageSize" yaml:"bufferPageSize"`
-	EnableCache          bool `json:"enableCache" yaml:"enableCache"`
-	ResendInterval       int  `json:"resendInterval" yaml:"resendInterval"`
-	CleanCacheAtStop     bool `json:"cleanCacheAtStop" yaml:"cleanCacheAtStop"`
+	MemoryCacheThreshold int          `json:"memoryCacheThreshold" yaml:"memoryCacheThreshold"`
+	MaxDiskCache         int          `json:"maxDiskCache" yaml:"maxDiskCache"`
+	BufferPageSize       int          `json:"bufferPageSize" yaml:"bufferPageSize"`
+	EnableCache          bool         `json:"enableCache" yaml:"enableCache"`
+	ResendInterval       int          `json:"resendInterval" yaml:"resendInterval"`
+	CleanCacheAtStop     bool         `json:"cleanCacheAtStop" yaml:"cleanCacheAtStop"`
+	SinkSQLConf          *SinkSQLConf `json:"sinkSQLConf" yaml:"sinkSQLConf"`
```

Comment:
It is not just sink actually, there is also sql source. Currently, if we put it in global configuration, we may put it in etc/kuiper.yaml